src/main/javacc/copy.jjt

```diff
@@ -0,0 +1,251 @@
+options {
+  MULTI=true;
+  VISITOR=true;
+  NODE_DEFAULT_VOID=true;
+  STATIC=false;
+}
+
+PARSER_BEGIN(Copy)
+
+package com.google.cloud.spanner.pgadapter.parsers.copy;
+
+public class Copy {
+
+  public static void parse(String stmt, CopyTreeParser.CopyOptions options) throws Exception {
+    java.io.InputStream stmtStream = new java.io.ByteArrayInputStream(stmt.getBytes());
+    Copy tree = new Copy(stmtStream);
+    try {
+      ASTStart node = tree.Start();
+      CopyVisitor v = new CopyTreeParser(options);
+      node.jjtAccept(v, null);
+    } catch (Exception e) {
+      System.err.println(e.getMessage());
+      e.printStackTrace();
+      throw e;
+    }
+  }
+
+}
+
+PARSER_END(Copy)
+
+SKIP :
+{
+  " "
+| "\t"
+| "\n"
+| "\r"
+| <"//" (~["\n","\r"])* ("\n"|"\r"|"\r\n")>
+| <"/*" (~["*"])* "*" (~["/"] (~["*"])* "*")* "/">
+}
+
+TOKEN : /* KEYWORDS */
+{
+  <K_STDIN: "STDIN">
```

Comment:
This does not include the `PROGRAM` keyword that is supported from [PostgreSQL 9.3 and later](https://www.postgresql.org/docs/9.3/sql-copy.html). That seems like a valid choice, but I think it would be good to document that both for users and for ourselves.

Reply:
We are currently only supporting a subset of the COPY functionality.  I shared an internal design doc with details.